src/test/resources/io/gravitee/policy/cache/integration/cacheV4.json

```diff
@@ -55,7 +55,8 @@
                         "scope": "API",
                         "cacheName": "dummy-cache",
                         "key": "integration-test-cache",
-                        "methods": ["GET"]
+                        "methods": ["GET"],
+                        "responseCondition": "{#upstreamResponse.status == 200}"
```

Comment:
Could you try with another condition ? So we are sure we test the condition and not the fallback on `is2xx` ?

Reply:
this condition is a fallback for all existing tests, but I added a new one shouldNotUseCache_BackendStatus503 that EL condition fail on
the is2xx is never tested though. I would need to add another API without responseCondition to test it, but that would be messy, because I would need to add another abstract test class. I checked that @DeployApi does not work on the method level. WDYT?